templates/sfm/countries.html

```diff
@@ -63,11 +64,11 @@ <h4><strong><i class="fa fa-fw fa-search"></i> {% trans "Browse data" %}</strong
                             </div>
                             <div class="col-xs-12 col-sm-8">
                                 <h4><strong><i class="fa fa-fw fa-clock-o"></i> {% trans "Collections" %}</strong></h4>
-                                <a href="{% url 'search' %}?selected_facets=country_ss_fct_exact:Egypt&selected_facets=organization_adminlevel1_ss_fct_exact:Cairo+Governorate&selected_facets=organization_classification_ss_fct_exact:Police&entity_type=Person&entity_type=Organization&entity_type=Violation">
-                                    <p>{% trans "Police in Cairo" %}</p>
+                                <a href="{% url 'search' %}?selected_facets=country_ss_fct_exact%3AEgypt&selected_facets=organization_parent_name_ss_fct_exact%3ACairo+Governorate+Security+Directorate&entity_type=Organization&Organization_rows=20">
+                                    <p>{% trans "Police in Cairo Governorate" %}</p>
```

Comment:
Do we have plans for translating these new tags?

Reply:
Assuming that they will appear in `/rosetta`, we can translate them once deployed.